src/components/tutorial-meta/components/interactive-lab-button/index.tsx

```diff
@@ -14,9 +16,13 @@ export default function InteractiveLabButton() {
   const buttonText = `${ctx.active ? 'Hide' : 'Show'} Terminal`
 
   return (
-    <button className={s.labButton} onClick={() => ctx.setActive(!ctx.active)}>
-      <IconTerminalScreen16 className={s.terminalIcon} />
-      <span>{buttonText}</span>
-    </button>
+    <Button
+      color="primary"
+      text={buttonText}
+      onClick={() => ctx.setActive(!ctx.active)}
+      size="medium"
+      iconPosition="leading"
+      icon={<IconTerminalScreen16 />}
+    />
```

Comment:
These values for `color`, `iconPosition`, and `size` are the default values, would you like to omit them?

Reply:
Yes, `disabled` also doesn't have elevation (essentially elevation is only for `default` + `hover` :)

Reply:
Oh rad! ya I'll remove those :)